metrics/interface/src/main/assembly/resources/conf/iotdb-metric.yml

```diff
@@ -18,11 +18,12 @@
 #
 
 # whether enable the module
-enableMetric: false
+enableMetric: true
```

Comment:
Please set default value as false because this will influence iotdb-server

Reply:
Agree, I'll fix this